scala3doc/src/dotty/dokka/tasty/TypesSupport.scala

```diff
@@ -68,10 +68,14 @@ trait TypesSupport:
     case List(single) => single
     case other => other.reduce((r, e) => r ++ texts(", ") ++ e)
 
-  private def isRepeated(tpeAnnotation: Term) =
+  // weird name fot type-erasure purposes
+  private def isRepeatedTerm(term: Term) =
     // For some reason annotation.tpe.typeSymbol != defn.RepeatedParamClass
     // annotation.tpe.typeSymbol prints 'class Repeated' and defn.RepeatedParamClass prints 'class <repeated>'
-    tpeAnnotation.tpe.typeSymbol.toString == "class Repeated"
+    term.tpe.typeSymbol.toString == "class Repeated"
```

Comment:
Is there a better way to check if type is Repeated? `toString` output is not a stable API

Reply:
I think we discussed it at least once at the beginning of scala3doc. There was a bug in TASTY that didn't allow us to simply check if symbol is `defn.ReapedParamClass`, maybe it's fixed now (?)

Reply:
```scala
  private def isRepeated(typeRepr: TypeRepr) =
    typeRepr match
      case t: TypeRef => t.name == "<repeated>" && t.qualifier.match
        case ThisType(tref: TypeRef) if tref.name == "scala" => true
        case _ => false
      case _ => false
```
is the best solution I can come up with